### PR TITLE
add reviewing as a possible contribution

### PR DESCRIPTION
**Pull request by zouharvi**
As was suggested during our chat, this adds *reviewing* as one of the possible contributions. Currently the pipeline is:

1. Issue is created (by any contributor)
2. A contributor picks up this issue, writes a draft of the article and files a PR
3. Adam reviews the article (+iterates)
4. Several iterations of changes are made be the original contributor
5. Adam merges the PR

This obviously does not scale and Adam becomes the bottleneck. Having someone else in place of Adam on step 3 would reduce the workload on him and he could be involved only in step 5 for final quality check.

I did not include this justification in the contributing file to keep it brief but believe that it'd be a step in the right direction (possibly with different wording) because a newcomer may think that reviewing is only reserved for "core maintainers".

Quality of the merged articles would not decrease because Adam would still read the final article at least once (though something could be done about that as well (democratization :slightly_smiling_face: )).

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).
- [x] I have made sure that the URL is not already in use.
- [x] I have cross-linked relevant articles.
- [x] I have used the UK spelling.
- [x] I have kept consistency with the structure of sister articles in the same directory.

### CODE REVIEW

**Comment by zouharvi** [comment]
Sorry to spam you with PRs this weekend. :slightly_smiling_face:

**Comment by cefoo** [comment]
Yes, I think it's a great idea to specify this in the Contributing guidelines!
I'll add @bittlingmayer as a reviewer of this PR.

> Sorry to spam you with PRs this weekend. 🙂

Not at all, THANK YOU!⭐️

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,19 +12,23 @@ Machine Translate is **created and edited by [contributors like you!](contributi
...
 ### Creating and editing articles
 
 To create or edit an article, [**create**](https://github.com/machinetranslate/machinetranslate.org/new/master) or [**edit**](https://github.com/machinetranslate/machinetranslate.org) and send us a [pull request](https://github.com/machinetranslate/machinetranslate.org/pulls?q=is%3Apr).
 
-> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more
+> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more.
+
+### Reviewing articles
+
+You can also contribute by reviewing other contributors' [open pull requests](https://github.com/machinetranslate/machinetranslate.org/pulls?q=is%3Aopen+is%3Apr) via comments and iterate with them to improve the articles.
```
Comment (bittlingmayer) [inline CONTRIBUTING.md:23]: Maybe we can remove "via comments and iterate with them to improve the articles."

(I'm just trying to keep sentences short.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,19 +12,23 @@ Machine Translate is **created and edited by [contributors like you!](contributi
...
 
-
 ### Creating and editing articles
 
 To create or edit an article, [**create**](https://github.com/machinetranslate/machinetranslate.org/new/master) or [**edit**](https://github.com/machinetranslate/machinetranslate.org) and send us a [pull request](https://github.com/machinetranslate/machinetranslate.org/pulls?q=is%3Apr).
 
-> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more
+> See GitHub’s article [*Creating a pull request*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) for more.
+
+### Reviewing articles
```
Comment (bittlingmayer) [inline CONTRIBUTING.md:21]: Maybe it should be

> ### Reviewing creation and edits

or just 

> ### Reviewing edits

or 

> ### Reviewing new and edited articles

Not sure.

**Review by bittlingmayer — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (zouharvi) [reply CONTRIBUTING.md]: Removed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (zouharvi) [reply CONTRIBUTING.md]: Changed to `reviewing proposed changes`.

**Pull request merged**